include/oneapi/dpl/pstl/omp/parallel_scan.h

```diff
@@ -0,0 +1,123 @@
+namespace oneapi
+{
+namespace dpl
+{
+namespace __omp_backend
+{
+
+template <typename _Index>
+_Index
+__split(_Index __m)
```

Comment:
I moved those functions to the given file, but I did not update the TBB implementation. It looks like it might need to be restructured a little to afford that.